packages/statemanager/src/statefulVerkleStateManager.ts

```diff
@@ -677,6 +680,7 @@ export class StatefulVerkleStateManager implements StateManagerInterface {
         canonicalValue = ZEROVALUE
       }
 
+      this._debug(`computed ${computedValue} canonical ${canonicalValue}`)
```

Comment:
```suggestion
```

I think this is an extra leftover debug log from work I was doing with the access witness cache.